native/packages/react-native-bpk-component-animate-height/src/BpkAnimateHeight.js

```diff
@@ -72,9 +95,12 @@ class BpkAnimateHeight extends React.Component {
     const toValue = expanded ? height : COLLAPSED_HEIGHT;
     const delay = expanded ? expandDelay : collapseDelay;
 
-    Animated.timing(this.height, { toValue, duration, delay }).start(
-      onAnimationComplete,
-    );
+    this.adjustChildHeight(height, () => {
+      Animated.timing(this.height, { toValue, duration, delay }).start(() => {
+        this.adjustChildHeight.reset();
+        if (onAnimationComplete) onAnimationComplete();
```

Comment:
Can we have braces here? I'm surprised prettier doesn't do this actually

Reply:
sure